pkg/storage/mock_helpers_test.go

```diff
@@ -0,0 +1,130 @@
+package storage_test
```

Comment:
Do we want to add these helpers to the `test/framework/framework.go` to be re-usable accross all test suites?

Reply:
I don’t know… as mentioned elsewhere, (with huge disclaimers) the necessity of this seems to be a code smell to me. From a quick look, it seems that many other packages have the mocked call sequences smaller, so they don’t need this as urgently.

The composable `inOrder` helper does seem possibly useful in general, or at least it does not clearly belong into `pkg/storage`. If any other test package needs the `storeMock`-based helper abstractions, although it accesses `containers/storage` via the `pkg/storage` interface, maybe that test package should instead use a `pkg/storage` mock instead of a `containers/storage` mock with the `storeMock`-based helpers. (Then again, end-to-end testing can genuinely be useful.)

(Also, practically, right now most of CRI-O does not build on macOS and at least this week I can’t spend much time changing that nor changing my development setup, so it is cumbersome for me to modify tests in most of the other packages. And I originally wanted to clean up c/image callers instead of redesigning tests.)